cmd/spec/main.go

```diff
@@ -86,9 +86,9 @@ func main() {
 
 	jsonB, err := json.MarshalIndent(doc, "", "  ")
 	checkErr(err)
-	err = ioutil.WriteFile("./api/openapi.gen.json", jsonB, 0644) // #nosec G306
+	err = ioutil.WriteFile("./api/openapi.json", jsonB, 0644) // #nosec G306
```

Comment:
the `.gen` suffix chamges the embed mechanism, the consumers seek for `openapi.json`, not `opemapi.gen.json`

Reply:
I was hoping for some alias, maybe symlink would work. It's very practical to have all generated files marked this way, my IDE even warns me about the fact that the file was likely generated.

However, I am fine with this. But the problem is this change was not reflected in `Makefile` so several targets do not work anymore. Can you either fix the `Makefile` or possibly try symlink or some kind of HTTP alias to make it work too?